pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -633,14 +638,24 @@ const (
 // InitLedCmd can use different LEDs in /sys/class/leds
 // Disable existing trigger
 // Write "none" to /sys/class/leds/<ledName>/trigger
-func InitLedCmd(ledName string) {
+// If comma-separated list try until one is found.
+// Returns the led which worked.
+func InitLedCmd(ledName string) string {
 	log.Functionf("InitLedCmd(%s)", ledName)
-	triggerFilename := fmt.Sprintf("/sys/class/leds/%s/trigger", ledName)
-	b := []byte("none")
-	err := ioutil.WriteFile(triggerFilename, b, 0644)
-	if err != nil {
-		log.Error(err, triggerFilename)
+	// If there are multiple, comma-separated ones, find one which works
+	leds := strings.Split(ledName, ",")
+	for _, led := range leds {
+		triggerFilename := fmt.Sprintf("/sys/class/leds/%s/trigger", led)
+		b := []byte("none")
+		err := ioutil.WriteFile(triggerFilename, b, 0644)
```

Comment:
Ideally we should fail if the trigger file doesn't exist. However, I think there is no problem here to change file mode to 0600 to make yetus happy since in real-world the file should be always there (created by led platform driver).

PS: ioutil is deprecated by I think Milan's PR already took care of this.